infra/docker/ci/Dockerfile

```diff
@@ -71,4 +71,15 @@ RUN PROTOC_ZIP=protoc-${PROTOC_VERSION}-linux-x86_64.zip && \
     go get gopkg.in/russross/blackfriday.v2 && \
 	  git clone https://github.com/istio/tools/ && \
 	  cd tools/cmd/protoc-gen-docs && \
-	  go build && mkdir -p $HOME/bin && cp protoc-gen-docs $HOME/bin
\ No newline at end of file
+	  go build && mkdir -p $HOME/bin && cp protoc-gen-docs $HOME/bin
+
+# Install AZ CLI
+RUN curl -sL https://aka.ms/InstallAzureCLIDeb | bash
+
+# Install kubectl
+RUN apt-get install -y kubectl
+
+# Install helm
```

Comment:
not sure if the feast-ci image already has kubectl and/or helm when it is running tests, so it's possible these installs are not needed. Also lmk if there's a better place to install the azure CLI

Reply:
I think its fine, but should the version be pinned?

Reply:
the version for which of these?

Reply:
I'd pin both kubectl and helm. kubectl is [only compatible within one minor version](https://kubernetes.io/docs/setup/release/version-skew-policy/#kubectl) so it is conceivable that one day this will suddenly stop working with our k8s clusters.